src/storage/query/ScanEdgeProcessor.cpp

```diff
@@ -67,7 +68,7 @@ void ScanEdgeProcessor::process(const cpp2::ScanEdgeRequest& req) {
         // only return data within time range [start, end)
         EdgeVersion version = folly::Endian::big(NebulaKeyUtils::getVersion(key));
         int64_t ts = std::numeric_limits<int64_t>::max() - version;
-        if (ts < startTime || ts >= endTime) {
+        if (FLAGS_enable_multi_versions && (ts < startTime || ts >= endTime)) {
```

Comment:
A question, if FLAGS_enable_multi_versions == false, here should be return all versions data? right?

Reply:
Yeah, we will return all versions. When `FLAGS_enable_multi_versions` is false, there should be only one version.